src/main/java/org/ossgang/commons/observables/DispatchingObservableValue.java

```diff
@@ -48,10 +48,11 @@ protected DispatchingObservableValue(T initial) {
     @Override
     public Subscription subscribe(Observer<? super T> listener, SubscriptionOption... options) {
         Set<SubscriptionOption> optionSet = new HashSet<>(Arrays.asList(options));
+        Subscription subscription = super.subscribe(listener, options);
         if (optionSet.contains(FIRST_UPDATE)) {
             Optional.ofNullable(lastValue.get()).ifPresent(listener::onValue);
```

Comment:
Can there be any problem, if the the listener::onValue throws and thus the subscription is not returned but the listener is subscribed to super ...?

Reply:
very good point! :slightly_smiling_face: 
Maybe then I take the opportunity to also fix the #42 .. so I try to use the dispatching mechanism + error deflection of the usual dispatch in a blocking way

Reply:
If the first update consumption fails I think we can still go ahead with the subscription. If you like it, I'll add a test and update docs :wink: